public/app/features/plugins/sql/components/query-editor-raw/QueryValidator.tsx

```diff
@@ -79,7 +79,7 @@ export function QueryValidator({ db, query, onValidate, range }: QueryValidatorP
     <>
       {state.loading && (
         <div className={styles.info}>
-          <Spinner inline={true} size={12} /> Validating query...
+          <Spinner inline={true} size="sm" /> Validating query...
```

Comment:
12px would be size xs. Did you change it to sm (14px) on purpose?

Reply:
sure didn't, just a mistake 😄 

great spot! 🚀